molecule/instance-name-test/converge.yml

```diff
@@ -60,7 +60,7 @@
       apiRoBi: "rbac.authorization.k8s.io/v1"
       apiServ: "v1"
       apiSvcA: "v1"
-      apiIngr: "networking.k8s.io/v1beta1"
+      apiIngr: "networking.k8s.io/{{ 'v1' if (lookup('k8s', kind='Ingress', api_version='networking.k8s.io/v1', errors='ignore') is iterable) else 'v1beta1' }}"
```

Comment:
This keeps backwards compatibility with 1.18 for the time being as v1beta1 is the fallback right?

Reply:
correct... if that lookup failed to find "v1" that if-stmt will fallback to v1beta1 (which is what we used before - so this is backwards compatible).